TA Signals/StocksDataFetcher.swift

```diff
@@ -16,12 +16,16 @@ class StocksDataFetcher {
     // The variable stocks contains an array of Stock objects
     @Published var stocks = [Stock]()
     
+    var _feature_flag_json_includes_date: Bool = false
+    
     // Classes in Swift do not have memberwise initializers ( like Structs do ) so we need to declare our own:
     init() {
-        // Original:
-//        let url = URL(string: "https://raw.githubusercontent.com/iamgabrielma/Python-for-stock-market-analysis/main/testData/2021-06-25-rsi.json")!
-        // Alternative, with new fields:
-        let url = URL(string: "https://raw.githubusercontent.com/iamgabrielma/Python-for-stock-market-analysis/main/testData/2021-06-25-rsi.json")!
+        
+        var url = URL(string: "https://raw.githubusercontent.com/iamgabrielma/Python-for-stock-market-analysis/main/testData/2021-06-25-rsi.json")!
```

Comment:
It looks like this is here for testing, your current changes, but wanted to drop this suggestion.  The URL you are using is unchanged, it is just the file name that is changing, so you could create a new file/class that contained static variables like this:

struct AppConstants {
    static let baseURL = "https://raw.githubusercontent.com/iamgabrielma/Python-for-stock-market-analysis/main/testData/"
}

Then you could rewrite these lines into something like:
 
let fileName = _feature_flag_json_includes_date ? "2021-07-27-rsi.json" : "2021-06-25-rsi.json"'
let url = URL(string: AppConstants.baseURL + fileName)!

This makes it easier to read, but more importantly if you ever had to update the base url for where this data is located you only need to update it in one place.  You could go around the rest of your program and insert AppConstants.baseURL every where the test data url appears.  Then update it all in one place.  In a large app like WPiOS that is crucial when updating endpoints and written constants

Reply:
This is a great idea!